_includes/header.html

```diff
@@ -37,6 +35,16 @@
         </li>
         <li><a href="{{ site.baseurl }}/blog">Blog</a></li>
       </ul>
+      <ul class="nav navbar-nav navbar-right">
```

Comment:
Subjective: I had some trouble finding the Project drop-down menu, which is particularly true for wide monitors. Not sure that I would do right alignment here.